packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.tsx

```diff
@@ -140,25 +183,42 @@ export class Coachmark extends BaseComponent<ICoachmarkProps, ICoachmarkState> {
   public componentDidMount(): void {
     // If we have already accessed the width and height
     // We do not wan't to do it again.
+    const entityInnerHostRect = this._entityInnerHostElement.offsetWidth;
```

Comment:
This is never used.

Reply:
great catch, yeah this was left over gunk from something i was working on.